corehq/apps/locations/fixtures.py

```diff
@@ -60,6 +60,11 @@ def should_sync_locations(last_sync, location_db, restore_user):
 class LocationFixtureProvider(object):
     id = 'commtrack:locations'
```

Comment:
yea that should work. it will just mean that for any commtrack fixture requested by cloudcare it will generate both of these even if they aren't required.